### PR TITLE
Update foreman package dependencies

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -569,6 +570,7 @@ Requires: %{?scl_prefix_ror}rubygem(execjs) < 3.0
 Requires: %{?scl_prefix_ror}rubygem(uglifier) >= 1.0.3
 Requires: %{?scl_prefix_ror}rubygem(sass-rails) >= 5.0
 Requires: %{?scl_prefix_ror}rubygem(sass-rails) < 6.0
+Requires: %{?scl_prefix_ror}rubygem(sass-rails) < 6
```
Comment (evgeni) [inline packages/foreman/foreman/foreman.spec]: that half-duplicates the line before?

File: packages/foreman/foreman/foreman.spec [hunk 2]
```diff
@@ -546,8 +547,8 @@ Requires: npm(webpack-stats-plugin) < 1.0.0
 # end package.json devDependencies Requires
 
 # start package.json dependencies Requires
-Requires: npm(@theforeman/vendor) >= 0.1.0
-Requires: npm(@theforeman/vendor) < 1.0.0
+Requires: npm(@theforeman/vendor) >= 1.4.0
```
Comment (evgeni) [inline packages/foreman/foreman/foreman.spec]:
```
[2019-08-22T15:43:31.263Z]     package: foreman-assets-1.24.0-0.2.develop.20190822153445git8d250fa.el7.noarch from downloaded_rpms
[2019-08-22T15:43:31.263Z]       unresolved deps:
[2019-08-22T15:43:31.263Z]          npm(@theforeman/vendor) >= 0:1.4.0
```

I guess that just wasn't mashed yet?

File: packages/foreman/foreman/foreman.spec [hunk 3]
```diff
@@ -339,6 +339,7 @@ BuildRequires: %{?scl_prefix_ror}rubygem(execjs) < 3.0
 BuildRequires: %{?scl_prefix_ror}rubygem(uglifier) >= 1.0.3
 BuildRequires: %{?scl_prefix_ror}rubygem(sass-rails) >= 5.0
 BuildRequires: %{?scl_prefix_ror}rubygem(sass-rails) < 6.0
+BuildRequires: %{?scl_prefix_ror}rubygem(sass-rails) < 6
```
Comment (evgeni) [inline packages/foreman/foreman/foreman.spec]: that half-duplicates the line before?

File: packages/foreman/foreman/foreman.spec (hunk 1 above)
Comment (ekohl) [reply packages/foreman/foreman/foreman.spec]: Yes, this is a side effect of the pinning (https://github.com/theforeman/foreman/blob/7c26a522e240798e01ec8f9cec3a73b7cc683018/bundler.d/assets.rb#L8):
```ruby
gem 'sass-rails', '~> 5.0', ((RUBY_VERSION < '2.4') ? '< 5.0.8' : '< 6')
```

Would have been more logical if it was:
```ruby
gem 'sass-rails', '>= 5.0', ((RUBY_VERSION < '2.4') ? '< 5.0.8' : '< 6.0')
```

Comment (ekohl) [reply packages/foreman/foreman/foreman.spec]: https://github.com/theforeman/foreman/pull/6996

**Pull request merged**